concent_api/core/validation.py

```diff
@@ -106,6 +106,23 @@ def validate_list_of_identical_task_to_compute(list_of_task_to_compute: List[mes
     return True
 
 
+def validate_list_of_identical_report_computed_task(list_of_report_computed_task: List[message.tasks.ReportComputedTask]):
```

Comment:
By adding `*` in front of the list you can easily make a function that takes variable number of arguments (not sure if the type annotation is still allowed in that case):
``` python
def validate_list_of_identical_report_computed_task(*list_of_report_computed_task: List[message.tasks.ReportComputedTask]):
```

Then you won't have to use a list when you call it:
``` python
identical_report_computed_task = validate_list_of_identical_report_computed_task(
    report_computed_task,
    deserialize_message(subtask.report_computed_task.data.tobytes()),
)
```

Reply:
It's working exactly as you wrote but Concent code is already adjusted to this kind parametrs forwarding to this function (already function merged with function to comparing TaskToCompute messages) and I think might be too much changes to do.

Reply:
I agree with @kbeker , I am not a fan of using `*` or `**` unless it is giving a big advantage and can save code. We use it nicely in cluster tests in one place, but here it would not improve the code much in my opinion.

Reply:
Sure. It was just a suggestion. I pointed it out because I would probably do it myself in this particular case and I wasn't sure if @kbeker considered this feature here.

But I agree that it should be used sparingly. For me it's a nice feature of cases where I know that a function will most of the time be used with short lists of constant length where each element is created separately by the caller.